docs/user/custom-resources/ratelimit/04-00-ratelimit.md

```diff
@@ -0,0 +1,64 @@
+| **local.buckets**                     | **NO**   | List of additional rate limit buckets for requests. <br/>Each bucket must specify either a `path` or `headers`. <br/>For each request matching the bucket's criteria, a single token is consumed. If a token is available, the request is allowed. If no tokens are available, the request is rejected with status code `429`.                                                                     |
+| **local.buckets.path**                | **NO**   | Specifies the path to be rate limited starting with `/`. <br/>For example, `/foo`.                                                                                                                                                                                                                                                                                                                 |
+| **local.buckets.headers**             | **NO**   | Specifies the request headers to be rate limited. The key is the header name, and the value is the header value. All specified headers must be present in the request for this configuration to match. For example, `x-api-usage: BASIC`.                                                                                                                                                          |                                                                                                                                                                                    
+| **local.buckets.maxTokens**           | **YES**  | The maximum tokens that the bucket can hold. This is also the number of tokens that the bucket initially contains.                                                                                                                                                                                                                                                                                 |
+| **local.buckets.tokensPerFill**       | **YES**  | The number of tokens added to the bucket during each fill interval.                                                                                                                                                                                                                                                                                                                                |
+| **local.buckets.fillInterval**        | **YES**  | The fill interval that tokens are added to the bucket. <br/>During each fill interval, `tokensPerFill` are added to the bucket. The bucket will never contain more than `maxTokens` tokens. The `fillInterval` must be greater than or equal to 50ms to avoid excessive refills.                                                                                                                   |
```

Comment:
```suggestion
| **local.buckets.fillInterval**        | **YES**  | The fill interval that tokens are added to the bucket. <br/>During each fill interval, `tokensPerFill` are added to the bucket. The bucket cannot contain more than `maxTokens` tokens. The `fillInterval` must be greater than or equal to 50ms to avoid excessive refills.                                                                                                                   |
```